nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -256,9 +257,23 @@ BtNavigator::initializeGoalPose()
 void
 BtNavigator::onGoalPoseReceived(const geometry_msgs::msg::PoseStamped::SharedPtr pose)
 {
-  nav2_msgs::action::NavigateToPose::Goal goal;
-  goal.pose = *pose;
-  self_client_->async_send_goal(goal);
-}
+  auto is_action_server_ready =
+    self_client_->wait_for_action_server(std::chrono::seconds(5));
+  if (!is_action_server_ready) {
+    RCLCPP_ERROR(client_node_->get_logger(),
+      "NavigateToPose action server is not available."
+      " Is the initial pose set?");
+    return;
+  }
+
+  nav2_msgs::action::NavigateToPose::Goal navigation_goal_;
+  navigation_goal_.pose = *pose;
 
+  // Enable result awareness by providing an empty lambda function
+  auto send_goal_options =
+    rclcpp_action::Client<nav2_msgs::action::NavigateToPose>::SendGoalOptions();
+  send_goal_options.result_callback = [](auto) {};
```

Comment:
Why add the empty lambda? this is for a topic, we have no result to give to anyone

Reply:
I guessed it was needed, but it apparently is not needed at all.

Reply:
> So was the issue that it was just silently not up yet?

nope it was because the action server name NavigationToPose -> navigation_to_pose. I thought you had a PR for this?